docs/installation.rst

```diff
@@ -308,11 +330,11 @@ Many of Pillow's features require external libraries:
 
         sudo pkg install python3
 
-    Prerequisites are installed on **FreeBSD 10 or 11** with::
+    Prerequisites are installed on **FreeBSD 11 or 12** with::
```

Comment:
Why update the versions here? Does libavif not work on FreeBSD 10?

Reply:
FreeBSD 10.4 was EoL in 2018 and libavif didn't even exist until 2020. I just tried running FreeBSD 10 in virtualization and it appears that https://pkg.freebsd.org/ no longer even hosts the 10.x binaries, so it's not even possible to get `pkg` running without hunting for a mirror that has kept the old versions around. If someone _does_ still have FreeBSD 10 running they would presumably be able to install libavif through ports. But the `pkg install` command below would not work.